hello.go

```diff
@@ -8,7 +8,6 @@ import (
 
 func main() {
 	// Hello world, the web server
-
 	helloHandler := func(w http.ResponseWriter, req *http.Request) {
```

Comment:
handler の ResponseWriter と Request の変数は慣習的に1文字を使うことが多いね

```suggestion
	helloHandler := func(w http.ResponseWriter, r *http.Request) {
```